frontend/src/app/shared/components/file-links/storage-file-list-item/storage-file-list-item.html

```diff
@@ -0,0 +1,32 @@
+<label
+  class="spot-list--item-action"
+>
+  <spot-checkbox
+    [checked]="selected"
+    (click)="changeSelection.emit()"
+  ></spot-checkbox>
+  <span
+    class="spot-icon spot-icon_{{fileLinkIcon.icon}} op-files-tab--icon op-files-tab--icon_{{fileLinkIcon.clazz}}"
```

Comment:
Is there a reason we're using `clazz` here? :D

Reply:
@b12f It is used to set the colors. What do you mean exactly?

Reply:
In discussion with @b12f: can stay like it is.